src/core/src/op/assign.cpp

```diff
@@ -137,6 +137,10 @@ bool Assign::has_evaluate() const {
 bool Assign::constant_fold(OutputVector& output_values, const OutputVector& inputs_values) {
```

Comment:
Can be removed as Base implementation will return false.

Reply:
done